### PR TITLE
New Medical Data Faker -- Updated Tests

### PR DESCRIPTION
**Pull request by vikramsinghchandel**
Added Code and iml file for Medical data faker  …
Contains following fakers
-- Hospital Name
-- Disease
-- Medicine
-- Symptoms

### CODE REVIEW

**Comment by vikramsinghchandel** [comment]
@codingricky 
Hi 
Fixed the test cases. Requesting for merge

File: src/test/java/com/github/javafaker/MedicalTest.java [hunk 1]
```diff
@@ -14,28 +14,28 @@ public void testMedicineName() {
...
 
     @Test
     public void testSymptomStatic() {
-        assertThat("Infertility (Female)", matchesRegularExpression("([\\w']+\\.?.\\(?[\\w']\\)?( )?){2,4}"));
+        assertThat("Infertility (Female)", matchesRegularExpression("([\\w']+\\.?.\\(?[\\w']\\)?( )?){2,15}"));
     }
 
     @Test
     public void testHospitalStatic() {
-        assertThat("LSU BOGALUSA MEDICAL CENTER (OUTPATIENT CAMPUS)", matchesRegularExpression("([\\w',]+\\.?.\\(?[\\w']\\)?( )?){2,4}"));
```
Comment (codingricky) [inline src/test/java/com/github/javafaker/MedicalTest.java]: Should this make a call to the faker class?

File: src/test/java/com/github/javafaker/MedicalTest.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    }
+
+    @Test
+    public void testSymptom() {
+        assertThat(faker.medical().symptoms(), matchesRegularExpression("([\\w']+\\.?.\\(?[\\w']\\)?( )?){2,5}"));
+    }
+
+
+    @Test
+    public void testSymptomStatic() {
```
Comment (codingricky) [inline src/test/java/com/github/javafaker/MedicalTest.java]: Should this make a call to the faker class?

File: src/test/java/com/github/javafaker/MedicalTest.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        assertThat("Infertility (Female)", matchesRegularExpression("([\\w']+\\.?.\\(?[\\w']\\)?( )?){2,15}"));
+    }
+
+    @Test
+    public void testHospitalStatic() {
+        assertThat("LSU BOGALUSA MEDICAL CENTER (OUTPATIENT CAMPUS)", matchesRegularExpression("([\\w',]+\\.?.\\(?[\\w']\\)?( )?){2,10}"));
```
Comment (codingricky) [inline src/test/java/com/github/javafaker/MedicalTest.java]: Should this make a call to the faker class?

File: src/test/java/com/github/javafaker/MedicalTest.java (hunk 2 above)
Comment (vikramsinghchandel) [reply src/test/java/com/github/javafaker/MedicalTest.java]: Hi @codingricky 
All the static test cases were kind of special cases (containing '(' OR ',')
But are now taken care of in regular test cases RegEx itself (eg. testHospital & testSymptom).
They can be removed, just added them for testing special cases.
Please suggest if i should remove them

**Comment by vikramsinghchandel** [comment]
All the static test cases were kind of special cases (containing '(' OR ',')
But are now taken care of in regular test cases RegEx itself (eg. testHospital & testSymptom).
They can be removed, just added them for testing special cases.
Please suggest if i should remove them

**Comment by vikramsinghchandel** [comment]
Hi @codingricky 
As suggested removed the static test cases.

**Pull request merged**